syncstorage/src/web/middleware/sentry.rs

```diff
@@ -148,6 +149,13 @@ where
                             return Ok(sresp);
                         }
                         report(&tags, event_from_error(apie));
+                    } else if let Some(tokenserver_error) = e.as_error::<TokenserverError>() {
```

Comment:
Do we want to do the same sort of logic (optionally checking to see if this is a "metric" error) for Tokenserver Errors?
(I guess we generate less non-actionable errors in TokenServer)
Also wondering if we should have the same basic pattern for both error types (have `is_reportable()`)

Reply:
Actually, I do think it'd be super useful to report BrowserID vs OAuth 4XX errors as metrics. I've added that, following the same `is_reportable()` type of pattern we are using for `ApiError`

Reply:
cool. We're duplicating the lines 148-151. I wonder if we can break those out so that we're not repeating things?

Reply:
Done!